src/components/EventFilterColumn.scss

```diff
@@ -1,15 +1,13 @@
 @media (min-width:851px) {
   .filter-bar__browse--dropdown-row {
-    display: -webkit-box;
     display: flex;
     justify-content: space-between;
-    -webkit-box-pack: justify;
+    box-pack: justify;
```

Comment:
We should reconsider whether `box-pack` should be used here. It dates back to the old spec.